### PR TITLE
Add bot account permission documentation

### PR DESCRIPTION
**Pull request by hanzala1234**

### CODE REVIEW

File: content/sre/gitops/bot-account.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Bot Account Permissions
+
+
+Stakater bot account will have following permissions
```
Comment (rasheedamir) [inline content/sre/gitops/bot-account.md:3]: too many extra empty lines

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-150-07b79f43`

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-150-6690fa88`

File: content/sre/gitops/bot-account.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+## GitOps Repo
+
+### Admin
+
+Bot account need to have admin permission in gitops repository as it's going to push to default branch of gitops repository. Bot account is used by tekton pipeline and push changes in gitops repository.
```
Comment (rasheedamir) [inline content/sre/gitops/bot-account.md]: redundant text

Comment (hanzala1234) [reply content/sre/gitops/bot-account.md]: what should have been there?

File: content/sre/gitops/bot-account.md (hunk 1 above)
Comment (hanzala1234) [reply content/sre/gitops/bot-account.md:3]: fixed

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-150-ebedd881`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-150-459f1775`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.95`

**Pull request merged**